queries.py

```diff
@@ -1,5 +1,15 @@
 from datetime import datetime
 from models import Node, Session, AVAILABILITY_TIMEOUT
+from sqlalchemy import func, distinct
+
+
+def count_active_nodes():
```

Comment:
suggest name for function: get_active_nodes_count_query()

Reply:
fixd